### ISSUE TITLE
Silencing several users at once

### ISSUE THREAD

**Issue by sampaccoud** [numerique-gouv/meet#605]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**
In a meeting with a lot of people I need to close the mike to several people.

**Describe the solution you'd like**
A button "close all microphones" to silence everybody but me.

**Comment by Moustique82** [comment]
You need to be able to manage host, co-host, and participant roles to define who has their microphones muted and who can unmute them.

You need to be able to enable this feature by default (mute for all participants except the host and co-hosts).

**Comment by cyberbaloo** [comment]
It would be interesting, because for people who have attention or hearing problems, if we hear interlocutors typing on their keyboard, we should be able to change them right away. I may seem severe... but it's so vital!
